### PR TITLE
Fixes #11171 - compute profiles nics & volumes select2 handling

### PR DESCRIPTION
**Pull request by tbrisker**

### CODE REVIEW

File: app/views/nic/_base_form.html.erb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <%= selectable_f f, :type, interfaces_types, {},
-                 :class => 'interface_type', :disabled => !f.object.new_record? %>
+                 :class => 'interface_type without_select2', :disabled => !f.object.new_record? %>
```
Comment (domcleal) [inline app/views/nic/_base_form.html.erb:2]: This one doesn't seem to work for me, I find the select2 for interfaces at the top of the edit modal isn't working (i.e. the element doesn't respond to clicks).  Not sure why it wouldn't as it looks identical to the others in this form.

Comment (tbrisker) [inline app/views/nic/_base_form.html.erb:2]: @domcleal This seems to work for me, what are you doing exactly that doesn't work (i.e. new/existing host/interface)? are there any errors in the js console? did you try clearing the cache (maybe you still use previous js version)?

Comment (domcleal) [inline app/views/nic/_base_form.html.erb:2]: Hmm, I can't reproduce it now.  Thanks for checking!

**Comment by domcleal** [comment]
Merged as 052d581, 83344af.  Thanks @tbrisker and @shlomizadok!

**Pull request closed without merging**